packages/richtext/src/utils/parse-tiptap-json.ts

```diff
@@ -0,0 +1,22 @@
+import { generateHTML } from '@tiptap/html';
+import { JSONContent, Extensions } from '@tiptap/core';
+import { debug } from '@sitecore-content-sdk/core';
+import StarterKit from '@tiptap/starter-kit';
+
+export const parseTiptapJSON = (content: JSONContent[], extensions: Extensions = []) => {
+  // prevent warns when StarterKit is passed in extensions
+  const nonDuplicateExtensions = Array.from(new Set([StarterKit, ...extensions]));
```

Comment:
It would be generally better to extend StarterKit and add new extensions. Unfortunately, extra extensions are not being applied up when doing so, and documentation on that part is lacking.
This current approach works though.

Reply:
Have you test current approach by providing customized extension, like `Heading.configure(...)`

Reply:
From the looks of it, `configure` is used to enable-disable extensions from `StarterKitOptions`. If docs are to be believed, extra extensions are added via `.extend()` call, that's what I tested:
https://tiptap.dev/docs/editor/extensions/custom-extensions/extend-existing

But then the custom extension wasn't picked up.